harness.c

```diff
@@ -0,0 +1,13 @@
+#include <stdint.h>
```

Comment:
It looks like we're not doing bounds checking here, so this bug: https://mayhem.forallsecure.com/rnshah9/elk/cli/-/defects/286085 appears to be in the harness itself. I would add bounds checking to the harness, (we want to find OOB reads in the underlying code, not the harness), and rename the harness to something that more closely resembles what's being fuzzed (in this case, js_eval).

Reply:
I'm a bit confused; in the defect that you linked, the backtraces in the test cases all point to within `js_eval`, not outside of it. So aren't the OOB reads happening in the source code?

What bounds need to be checked? I don't see any array accesses in the harness, either.

Reply:
When I initially looked at this it seemed as though you were passing an incongruent length parameter to js_eval() (the `~0U`, which seemed disingenous. Now after passing the size parameter instead I am still seeing the crash pop up, so I think this is legitimate. This also seems to be the case after looking at the declaration here: 
https://github.com/cesanta/elk/blob/9112ce0f7e5ecfefa176d3fd827b17c84b720c00/elk.c#L1387

Anyways, I did make some other changes, which you can find in this PR: https://github.com/rnshah9/elk/pull/1